dev/source/docs/using-gazebo-simulator-with-sitl.rst

```diff
@@ -136,7 +136,24 @@ Then find the file libgazebo_common.so.1, probably under `/usr/local/lib` or `/u
     Compiling Gazebo from source will not be necessary once this pull request gets merged:
     https://bitbucket.org/osrf/gazebo/pull-requests/2450/ardupilot-refactor-and-minor-improvements/diff
 
+Installing gazebo pre-built packages
+====================================
 
+Instead of building gazebo yourself you can instead install a prebuilt
+set of packages if you are running a suitable distro of Linux.
+
+Instructions for installing the "SASC" version of gazebo that works
+with ArduPilot SITL are here:
+
+  https://github.com/osrf/uctf/tree/master/doc/install_binary
+
+That will install gazebo in /opt/sasc, so you need to also do:
+
+::
+   export PATH=/opt/sasc/bin:$PATH
```

Comment:
better to do something like echo 'export PATH=/opt/sasc/bin:$PATH' >> ~/.bashrc or people will complain that is not working after a reboot !